components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -8825,6 +8861,74 @@ definitions:
           usage count of Scope
         example: 3
 
+#-----------------------------------------------------
+# The Shared Scope Usage resource
+#-----------------------------------------------------
+  SharedScopeUsage:
+    title: SharedScopeUsage
+    required:
+      - id
+      - name
+    properties:
+      id:
+        type: string
+        description: |
+          UUID of the Scope. Valid only for shared scopes.
+        example: 01234567-0123-0123-0123-012345678901
+      name:
+        type: string
+        description: |
+          name of Scope
+        example: apim:api_view
+      usedApiList:
+        description: |
+          API list which have used the shared scope
+        type: array
+        items:
+          $ref: '#/definitions/SharedScopeUsedAPIInfo'
+
+#-----------------------------------------------------
+# The API object where the shared scope is used
+#-----------------------------------------------------
+  SharedScopeUsedAPIInfo:
+    title: API object using shared scope
+    required:
+      - name
+      - context
+      - version
+    properties:
+      name:
+        type: string
+        example: CalculatorAPI
+      context:
+        type: string
+        example: CalculatorAPI
+      version:
+        type: string
+        example: 1.0.0
+      provider:
+        description: |
+          If the provider value is not given user invoking the api will be used as the provider.
+        type: string
+        example: admin
+      usedResourceList:
+        description: |
+          Resource list which have used the shared scope within this API
+        type: array
+        items:
+          $ref: '#/definitions/SharedScopeUsedAPIResourceInfo'
+
+  #-----------------------------------------------------
+  # The API resource object where the shared scope is used
+  #-----------------------------------------------------
+  SharedScopeUsedAPIResourceInfo:
+    title: API resource object using shared scope
+    properties:
+      target:
+        type: string
+      verb:
+        type: string
```

Comment:
shall we add examples here too?

Reply:
Fixed with 3550c6ac829d5e756dab682eab507f16e5616d0b